### PR TITLE
Add b2aipreview to appropriate dashboard tab

### PR DESCRIPTION
**Pull request by ibevers**
This pull request includes updates to the `src/dashboard.py` file to enhance the dataset structure preview page by integrating external content and improving the user interface. The most important changes include importing necessary libraries, defining a constant URL, and updating the `dataset_structure_preview_page` function to fetch and display HTML content.

### Enhancements to Dataset Structure Preview:

* [`src/dashboard.py`](diffhunk://#diff-06135492c180d831917894055e5bb06ed9c4f31998a88b446abd432d50f6a4c7R1-R10): Imported the `requests` library and `streamlit.components.v1` to enable fetching and displaying external HTML content.
* [`src/dashboard.py`](diffhunk://#diff-06135492c180d831917894055e5bb06ed9c4f31998a88b446abd432d50f6a4c7R1-R10): Defined the `B2AIPREVIEW_GITHUB_RAW_URL` constant to store the URL of the external HTML content.
* `dataset_structure_preview_page` function in `src/dashboard.py`: Updated to fetch HTML content from the defined URL and display it using `components.html`. Added error handling to display an error message if the content fails to load.

### CODE REVIEW

**Review by alexsigaras — CHANGES_REQUESTED** [review]
Hi @ibevers,

Could you please add the .html portion referenced in the `src/dashboard.py` as a static project resource rather than a relative link? We would like to ensure that all content displayed on this project is contained within and there are no external dependencies

**Comment by ibevers** [comment]
@alexsigaras I have integrated the html file locally, as well as the template directory structure. The ref to the directory structure refs my fork currently since there isn't a checksum associated with it yet on the source repo. This can be updated once this is merged so that everything is local to this repo.

File: src/dashboard.py [hunk 1]
```diff
@@ -55,7 +59,7 @@ def main():
     tab_functions = {
         "About": about_page,
         "Healthsheet": healthsheet_page,
-        "Study Dashboard": study_dashboard_page,
+        # "Study Dashboard": study_dashboard_page,
```
Comment (alexsigaras) [inline src/dashboard.py:62]:
```suggestion
        "Study Dashboard": study_dashboard_page,
```

File: src/dashboard.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
+import requests
+import os
+
 import streamlit as st
+
 from tabs.about import about_page
 from tabs.healthsheet import healthsheet_page
-from tabs.study_dashboard import study_dashboard_page
+# from tabs.study_dashboard import study_dashboard_page
```
Comment (alexsigaras) [inline src/dashboard.py:8]:
```suggestion
from tabs.study_dashboard import study_dashboard_page
```

**Review by alexsigaras — APPROVED** [review]
🚀

**Pull request merged**